appendices/extension_index.txt

```diff
@@ -50,8 +50,15 @@
 * <<extension_renderer, `KHR_RENDERER_BACKGROUND_IMAGE`>>:  the renderer supports a background image
 * <<extension_renderer, `KHR_RENDERER_DENOISE`>>:  the renderer supports denoising
 * <<object_types_sampler_image1D, `KHR_SAMPLER_IMAGE1D`>>: `image1D` sampler subtype is supported
+* <<extension_sampler_clamp_to_border_image1d, `KHR_SAMPLER_CLAMP_TO_BORDER`>>: `image1D` sampler wrapMode supports clampToBorder
 * <<object_types_sampler_image2D, `KHR_SAMPLER_IMAGE2D`>>: `image2D` sampler subtype is supported
+* <<extension_sampler_clamp_to_border_image2d, `KHR_SAMPLER_CLAMP_TO_BORDER`>>: `image2D` sampler wrapMode1, wrapMode2 support clampToBorder
 * <<object_types_sampler_image3D, `KHR_SAMPLER_IMAGE3D`>>: `image3D` sampler subtype is supported
+* <<extension_sampler_clamp_to_border_image3d, `KHR_SAMPLER_CLAMP_TO_BORDER`>>: `image3D` sampler wrapMode1, wrapMode2, wrapMode3 support clampToBorder
+* <<extension_sampler_image_clamp_to_border, `KHR_SAMPLER_IMAGE_CLAMP_TO_BORDER`>>:
```

Comment:
I see what you mean, this is indeed tricky as we have one extension for multiple samplers. I'd just keep this entry and primarily link to `image1d`, then link the bullets to the specific one.
I can simply make those changes if fine with you.